distributed/tests/test_client.py

```diff
@@ -6666,7 +6704,7 @@ async def f(stacklevel, mode=None):
     assert "Dask Performance Report" in data
     assert "x = da.random" in data
     assert "Threads: 4" in data
-    assert "No logs to report" in data
+    assert "INFO - Run out-of-band function" in data
```

Comment:
performance_report calls client.run_on_scheduler:
https://github.com/dask/distributed/blob/50aedd7d8ce76442076779b62a618c04af7dd19d/distributed/client.py#L5787-L5789

which now correctly logs to `distributed.scheduler` rather than `distributed.worker` so this log message will always show up in performance reports.

If this isn't desirable I could add a "get_monitor_count" handler to the scheduler that doesn't cause a message to be logged

Reply:
ok I'll leave it for now